code/__HELPERS/logging/silo.dm

```diff
@@ -0,0 +1,3 @@
+/proc/log_silo(text)
+	if (CONFIG_GET(flag/log_silo))
```

Comment:
We don't need a config for every single logging entry

Reply:
Made it so that silo logs are always created